pkg/util/mount/mount.go

```diff
@@ -25,7 +25,7 @@ import (
 	"k8s.io/kubernetes/pkg/util/exec"
 )
 
-type Interface interface {
+type Mounter interface {
```

Comment:
I reverted this change (renaming ```mount.Interface``` to ```mount.Mounter```) to avoid flame war in this PR. However even without composing objects out of two interfaces, naming interface ```Interface``` can lead to ambiguity.  Let's imagine:

```
package object

type Interface interface {
  GetPosition() Position
  Move()
}
```
and an extension of an object:
```
type Apple {
  object.Interface
  colour Colour
}
```

When I see ```apple.Interface``` I have no idea what this interface represents. I think it's much more readable if I see ```apple.Object``` or ```apple.ObjectInterface```.